README.md

```diff
@@ -111,6 +111,66 @@ When running on cluster node, you will also need to distribute this keytab, belo
 
 When Spark application is started, it will transparently track the execution plan of submitted SQL/DF transformations, parse the plan and create related entities in Atlas.
 
+Spark models vs Hive models
+====
+
+SAC classifies table related entities with two different kind of models: Spark / Hive.
+
+We decided to determine which models SAC should create based on whether Hive Hook can take care of table related entities or not. For Hive entities, SAC relies on HMS Atlas hook as the source of truth and expects HMS atlas hook to create it.
+
+As SAC cannot check whether the condition is properly made, SAC relies on precondition: if Spark session is connected to the "remote HMS" (via thrift), SAC assumes Hive Hook is already set up in remote HMS and Hive Hook will take care of entities creation.
+
+In other words, SAC assumes table entities are being created in Hive side and just refers these entities via object id if below conditions are true:
+
+* SparkSession.builder.enableHiveSupport is set
+* The value of "hive.metastore.uris" is set to non-empty
+
+For other cases, SAC will create necessary table related entities as Spark models.
+
+One exceptional case is HWC - for HWC source and/or sink, SAC will not create table related entities and always refer to Hive table entities via object id.
```

Comment:
Why is this an exceptional case? I think this is already covered by the condition that `SparkSession.builder.enableHiveSupport` is set and `hive.metastore.uris` is not empty

Reply:
That's not the case. Even it doesn't meet the condition (so we indicate tables as Spark table), source and sink tables for HWC are obviously Hive table.

Reply:
But can we use HWC if `SparkSession.builder.enableHiveSupport` is NOT set or `hive.metastore.uris` is empty?

Reply:
According to the README on HWC, it requires HS2 information instead of HMS, so I'd think twos are irrelevant.

https://github.com/hortonworks-spark/spark-llap/blob/master/README.md

Reply:
I don't think HWC can be used if any of those configurations are not set: `SparkSession.builder.enableHiveSupport` or `hive.metastore.uris`. So, I think we can remove this statement.

Reply:
At least document doesn't look like requiring that. I'll try to confirm whether the condition is true.